rules-tests/TypeDeclaration/Rector/ClassMethod/AddReturnArrayDocblockBasedOnArrayMapRector/Fixture/skip_if_else.php.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddReturnArrayDocblockBasedOnArrayMapRector\Fixture;
+
+final class SkipIfElse
+{
+    public function __construct(array $items)
```

Comment:
`__construct()` can't have `@return`, should use different method instead, eg: `run` or `process`